.circleci/config.yml

```diff
@@ -10,14 +10,17 @@ jobs:
       - checkout
 
       - run:
-          name: Install python 3.9
-          command: sudo apt-get install python3.9-venv
+          name: Install python 3.11
+          command: |
+            sudo add-apt-repository ppa:deadsnakes/ppa
```

Comment:
Is adding this repository required?

Reply:
Yes, as 3.11 is not available at least in this CircleCI runner (not sure where we can pick a more updated runner)